index.js

```diff
@@ -0,0 +1,74 @@
+const express = require("express");
+const app = express();
+const axios = require("axios");
+const pool = require("./database");
+require("dotenv").config();
+
+app.use(express.json());
+
+let port = process.env.PORT || 3000;
+
+async function universal_api_posts() {
+  try {
+    const data_1 = await pool.query(
+      `select discord_url from github_discord_url where id = $1`,
+      [`${db_id}`]
+    );
+    // console.log("ok");
+    // console.log(data_1.rows[0]);
+
+    const github_discord_url = data_1.rows[0];
+    // console.log(github_discord_url);
+
+    if (data_1.rows === null || data_1.rows >= 2) {
+      throw new Error("row doesn't exist");
+    }
+
+    var message = `${param1} just pushed a commit with message - '${param2}' to <${param3}>`;
```

Comment:
all this shouldn't happen here - only fetching from db & returning should happen within this function

Reply:
27 & onwards should happen within api function only